src/db/migrations.dat

```diff
@@ -0,0 +1 @@
+LAST_MIGRATION=1
```

Comment:
rebase your branch. I added `migrations.dat` to gitignore so we don't commit this.